Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -122,7 +124,9 @@ public function build(array $buildSubject): array
 
         // Ratepay specific Fingerprint
         $deviceFingerprint = $payment->getAdditionalInformation(AdyenPaymentMethodDataAssignObserver::DF_VALUE);
-        if ($deviceFingerprint && $this->adyenHelper->isPaymentMethodOfType($payment->getMethod(), Data::RATEPAY)) {
+
+        if (
+            $deviceFingerprint && str_contains($payment->getMethod(), self::RATEPAY)) {
```

Comment:
What would you think about moving the `RATEPAY` constant to the `PaymentMethods` helper as we already have some method name constants store there?